cromwell-backend/CHANGELOG.MD

```diff
@@ -0,0 +1,5 @@
+# Cromwell-Backend Change Log
+
+1.0:
+=====
+
```

Comment:
Yes, it is a separate entity which the full cromwell jar is but one consumer of it